compiler/modulegraphs.nim

```diff
@@ -28,7 +28,7 @@
 import ast, intsets, tables, options, lineinfos, hashes, idents,
   incremental, btrees, md5
 
-import ic / packed_ast
+# import ic / packed_ast
```

Comment:
Why comment this?

Reply:
to silence `UnusedImport` warning; I can also just remove this line if you prefer

Reply:
Maybe don't handle UnusedImport warning it in this PR, `IC` changed rapidly I guess.

Reply:
I mean it's easy enough to uncomment this line if it ever becomes needed :) 
also helps reduce risk of cyclic imports